tinyfl/scorer.py

```diff
@@ -23,7 +57,7 @@ def marginal_gain_scorer(weights, prev_scores, testloader):
     ]
 
 
-def multikrum_scorer(weights):
+def multikrum_scorer(weights: List[Mapping[str, Any]]):
```

Comment:
return type is list of floats
```suggestion
def multikrum_scorer(weights: List[Mapping[str, Any]]) -> List[float]:
```